### PR TITLE
fix(dedicated): fix subnet list and add modal text for ipv6 on vrack

### PR DESCRIPTION
**Pull request by mhelhali-soufien**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/additional-ipv6-feature` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-14302 #MANAGER-14351 #MANAGER-14306
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html [hunk 1]
```diff
@@ -9,71 +9,79 @@
...
-            data-oui-popover-scope="$ctrl"
-            data-oui-popover-template="vrack__assigned-ipv6__subrange_slaac_help.html"
-        ></button>
-        <script
-            type="text/ng-template"
-            id="vrack__assigned-ipv6__subrange_slaac_help.html"
+    <ul class="vrack__assigned-ipv6__sub-item">
+        <li
+            class="vrack__assigned-ipv6__subrange"
+            data-ng-repeat="bridged in $ctrl.bridgedSubranges track by $index"
```
Comment (qpavy) [inline packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html:15]: I thought there was only 1 bridged subrange ?

File: packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html [hunk 2]
```diff
@@ -9,71 +9,79 @@
...
-                data-translate-values="{ ipblock: $ctrl.ip.niceName, ip: subnet }"
+                class="vrack__assigned-ipv6__subrange__title"
+                data-ng-bind="::bridged.bridgedSubrange"
             ></span>
-            <oui-button
-                size="s"
-                variant="link"
-                ng-click="$ctrl.openDeleteSubnetModal(subnet)"
+            <span>
+                {{:: $ctrl.label.SLAAC }}
```
Comment (darsene) [inline packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html:22]:
```suggestion
               <span data-ng-bind="$ctrl.label.SLAAC"></span>
```

File: packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html [hunk 3]
```diff
@@ -9,71 +9,79 @@
...
+        <li
+            class="vrack__assigned-ipv6__subnet row"
+            data-ng-repeat="subnet in $ctrl.subnets track by $index"
+        >
+            <div
+                class="vrack__assigned-ipv6__subnet__title col-md-10 col-lg-10 col-xs-10"
             >
-                <span class="oui-icon oui-icon-bin"></span>
-            </oui-button>
+                <div>{{:: subnet.routedSubrange }}</div>
```
Comment (darsene) [inline packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html:70]:
```suggestion
                <div data-ng-bind=":: subnet.routedSubrange"></div>
```

**Review by darsene — CHANGES_REQUESTED** [review]

File: packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html [hunk 4]
```diff
@@ -9,71 +9,79 @@
...
+            </span>
+            <button
+                type="button"
+                class="oui-popover-button"
+                data-oui-popover="{{:: 'vrack__assigned-ipv6__subrange_slaac_help' | translate }}"
+                data-oui-popover-placement="right"
+                data-oui-popover-scope="$ctrl"
+                data-oui-popover-template="vrack__assigned-ipv6__subrange_slaac_help.html"
+            ></button>
+            <script
```
Comment (qpavy) [inline packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html:40]: Better to do a component no ?

File: packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html (hunk 1 above)
Comment (mhelhali-soufien) [reply packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html:15]: based on Api we can have many subrange just when we add ipv6 to vrack first one created by default

File: packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html (hunk 4 above)
Comment (mhelhali-soufien) [reply packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html:40]: like as other implementation of popover I implement this one but if you think is more better to create new component no problem

File: packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html (hunk 1 above)
Comment (qpavy) [reply packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html:15]: Ok weird but if it is possible from the API let's keep  it like this :)

**Review by qpavy — APPROVED** [review]

File: packages/manager/modules/vrack/src/vrack-associated-services/ipv6/slaac-help.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<p data-translate="vrack__assigned-ipv6__subrange_slaac_help"></p>
+<a
+    class="oui-link_icon"
+    aria-hidden="true"
```
Comment (darsene) [inline packages/manager/modules/vrack/src/vrack-associated-services/ipv6/slaac-help.html:4]: according to sonar analysis, remove `aria-hidden` attribute and replace it with `rel` attribute
```suggestion
   rel="noopener"
```

**Review by darsene — APPROVED** [review]

**Pull request merged**